### DISCUSSION TITLE
Grouping of Reading Lists for organization

### DISCUSSION THREAD

**Discussion by DieselTech** [gotson/komga#1534] [Ideas]
### Describe your suggested feature

Having some way to group up like reading lists would help make things less 'all over the place'. If I were to upload the entire reading list #[repo](https://github.com/DieselTech/CBL-ReadingLists) there would be no way to control where things go. It would just be a massive scroll bar and info dump. 

Not sure what is possible using the languages available to you, but something along the lines of folders would be nice to help sort this out and make it presentable. Maybe not directly folders, but like how gmail uses tags to group up convo's. Something like `[Marvel] [Events] [X-Men] Dawn of X.cbl` would get broken down into a UI view of:
```
|───Marvel 
|   ├───Events
|            └───X-Men
|                   └───Dawn Of X
```

Filenames would use what is in [] to display how the "folder" structure would be presented in the UI. Anything in [] would then be stripped out when looking at the folder view. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
There's already a request for tags for reading lists #869

**Comment by DieselTech** [comment]
I was really just using that as an example since traditional folders might not translate well. Heck I'd like the ability for it to just read from the file system of reading lists and do it that way. Could create a folder `ReadingLists` inside the config folder and could dump  the lists there for it to follow the structure.

**Comment by gotson** [comment]
Similar to #1544

**Comment by JJBassQue** [comment]
I am putting in another vote for this. I was imagining a way to create Categories for read list. I think this could add a fun element to read lists and entice people to want to use it. It could look just like series or read list element (with posters and title at the bottom), but when clicked it would show the read lists.

For example, I have multiple read lists like the ones below but it adds more repeated text to the page and it feels disorganized. Instead I could have a Category for each (Fav runs, Fav artists, Events, Sagas, etc.), and inside each one would be a bunch of read lists:
- Artist - Todd McFarlane
- Artist - Arthur Adams
- Artist - Marshall Rogers
- etc.
- Best runs - Captain America by Ed Brubaker
- Best runs - Immortal Hulk
- Best runs - X-Force by Remender
- Best runs - Vision by Tom King
- etc.
- Events - Infinity Gauntlet
- Events - Atlantic Attacks
- Events - Secret Invasion
- etc.
- Saga/Story arc - Thanos War
- Saga/Story arc - Vision origin
- Saga/Story arc - Warlock saga
- Favorite Wolverine covers
- Favorite Covers
- etc.
- First appearances Heroes
- First appearances Villains
- Important runs
- Important books
- etc.

Great work on everything btw, Komga is so much fun :)
